src/main/java/edu/cornell/kfs/module/purap/CUPurapConstants.java

```diff
@@ -268,4 +268,24 @@ public static final class Einvoice {
         public static final String ROOM_NUMBER_PREFIX = "Room ";
         public static final String VENDORS = "vendors";
     }
+    
+    public static final class JaggaerLinkMappingFieldNames {

```

Comment:
Should the constants from this nested class be moved to the CUPurapPropertyConstants class? Or would that interfere with the setup of the enum below?

Reply:
I was able to move them